content/content-strategy/manage-content-requests/create-content/content-production-template/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Content production template
+weight: 40
+description: Use the content production template to prepare, write and check for quality content.
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - toc.md
+  - cards.md
+  - using-head.md
+  - using.md
+  - resources-cards-head.md
+  - resources-cards.md
```

Comment:
Missing the global feedback partial, `- /_shared/feedback.md` Please add to the bottom of the `main` area.

Reply:
Trisha: done